lib/util/traffic-switch/traffic-switcher-i-token-fee-fetcher.ts

```diff
@@ -0,0 +1,72 @@
+import { TrafficSwitcher } from './traffic-switcher'
+import { ITokenFeeFetcher, TokenFeeMap } from '@uniswap/smart-order-router/build/main/providers/token-fee-fetcher'
+import { ProviderConfig } from '@uniswap/smart-order-router/build/main/providers/provider'
+
+type Address = string
+
+export class TrafficSwitcherITokenFeeFetcher extends TrafficSwitcher<ITokenFeeFetcher> implements ITokenFeeFetcher {
+  constructor(
+    experimentName: string,
+    implementationA: ITokenFeeFetcher,
+    implementationB: ITokenFeeFetcher,
+    aliasA: string,
+    aliasB: string,
+    pctTrafficSwitchToB: number,
+    pctSamplingToCompare: number
+  ) {
+    super(experimentName, implementationA, implementationB, aliasA, aliasB, pctTrafficSwitchToB, pctSamplingToCompare)
+  }
```

Comment:
This constructor can be ignored, since you are not passing any new information, the constructors are inherited as well.